account_analytic_default_account/README.rst

```diff
@@ -0,0 +1,74 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+Module name
+===========
+
+This module was written to extend the functionality of paying an invoice
+and allow you to get a default analytic account on counterpart move line
+when invoice's currency is not the same than company currency
+
+Installation
+============
+
+To install this module, you need to:
+
+* just install it !
+
+Configuration
+=============
+
+To configure this module, you need to:
+
+* go to Accounting / Analytic Accounting / Analytic Defaults
+* create analytic defaults and set an general account in it
+
+Usage
+=====
+
+To use this module, you need to:
+
+* Nothing
+
+For further information, please visit:
+
+* https://www.odoo.com/forum/help-1
+
+Known issues / Roadmap
+======================
+
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues <https://github.com/OCA/{project_repo}/issues>`_.
```

Comment:
Can you replace ```{project repo}``` with ```account-financial-tools```
+Bugs are tracked on `GitHub Issues <https://github.com/OCA/{project_repo}/issues>`_.